core/chains/evm/config/chain_specific_config.go

```diff
@@ -155,13 +157,16 @@ func setChainSpecificConfigDefaultSets() {
 	mainnet.eip1559DynamicFees = true // enable EIP-1559 on Eth Mainnet and all testnets
 	mainnet.linkContractAddress = "0x514910771AF9Ca656af840dff83E8264EcF986CA"
 	mainnet.minimumContractPayment = assets.NewLinkFromJuels(100000000000000000) // 0.1 LINK
+	mainnet.operatorFactoryAddress = "0x3e64cd889482443324f91bfa9c84fe72a511f48a"
```

Comment:
It looks like this is copied to ropsten/goerli/rinkeby too - should they overwrite blank instead?